tests/test_def.c

```diff
@@ -38,15 +38,45 @@ static void test_noreftracking() {
   upb_msgdef_unref(md, &md);
 }
 
+static char *upb_readfile(const char *filename, size_t *len) {
+  long size;
+  char *buf;
+  FILE *f = fopen(filename, "rb");
+  if(!f) return NULL;
+  if(fseek(f, 0, SEEK_END) != 0) goto error;
+  size = ftell(f);
+  if(size < 0) goto error;
+  if(fseek(f, 0, SEEK_SET) != 0) goto error;
+  buf = malloc(size + 1);
+  if(size && fread(buf, size, 1, f) != 1) goto error;
+  fclose(f);
```

Comment:
ditto, better yet can you share this utility function instead of reimplementing?